source/User_Guide/Settings/Subusers/index.html

```diff
@@ -20,7 +20,9 @@
 Manage Subusers
 {% endanchor %}
 
-To manage a subuser, click the name of the subuser you would like to manage. This will take you to the subuser’s profile page. From there, you can manage and see quite a bit of information about the subuser you selected.
+Subusers can be managed via [Subuser API](https://sendgrid.api-docs.io/v3.0/subuser-api) or by logging into your SendGrid account under Settings > Subuser Management.
```

Comment:
Oooh, I figured out the issue! This is one of the odd .html pages, so the link does need to be in the `<a href...>` form. When you get a chance, do you mind fixing this link, and the other broken one too (on line 31)